src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
```

Comment:
if you want to adjust the same  font-family to the whole body you should give class to your body. tag is less specified than class, so these styles can be easily blocked by the same properties set in classes.